jdbc/pom.xml

```diff
@@ -22,6 +22,7 @@
 		<module>immutables</module>
 		<module>jmolecules</module>
 		<module>jooq</module>
+		<module>howto</module>
```

Comment:
Care to retain alphabetical ordering? We should also fix `mybatis` order-wise.

Reply:
Done.